### ISSUE TITLE
v8.10.2 plugin not working

### ISSUE THREAD

**Issue by uploadx10** [GC-spigot/AdvancedEnchantments#2830]
### Describe the bug

Deleted the folder and run again, it can't run.

### How to reproduce

Deleted the folder and run again, it can't run.

### Screenshots / Videos

https://pastebin.com/0phgjkDT

### "/ae plinfo" link

plugin not working can not use

### Server Log

https://pastebin.com/0phgjkDT

**Comment by itsmemac** [comment]
Facing same issue with running latest update of AE on 1.18.2
**Version 8.10.2** : <https://www.toptal.com/developers/hastebin/aqiguqoviy.md> (Doesn't work)
**Version 8.9.24** : <https://www.toptal.com/developers/hastebin/aqetimogaj.md> (Works without any issues)
**Server version** : git-Paper-315 (MC: 1.18.2)
Tried running the plugin with all the other plugins as well as separate server only with AE so i dont think its issue with other plugins conflict
both gave the same error

**Comment by oORevanOo** [comment]
Have this issue too, same error code
also 1.18.2

**Comment by SirJben** [comment]
> ### Describe the bug
> Deleted the folder and run again, it can't run.
> 
> ### How to reproduce
> Deleted the folder and run again, it can't run.
> 
> ### Screenshots / Videos
> https://pastebin.com/0phgjkDT
> 
> ### "/ae plinfo" link
> plugin not working can not use
> 
> ### Server Log
> https://pastebin.com/0phgjkDT

Update to the latest version.

**Comment by itsmemac** [comment]
Issue has confirmed to be fixed in v8.10.3
Now it loads without any issues
```
[23:08:10 INFO]: [AdvancedEnchantments] Enabling AdvancedEnchantments v8.10.3
[23:08:11 INFO]: [AdvancedEnchantments] Successfully hooked into WorldGuard.
[23:08:11 INFO]: [AdvancedEnchantments] Successfully hooked into AureliumSkills.
[23:08:11 INFO]: [AdvancedEnchantments] Successfully hooked into PlaceholderAPI.
[23:08:11 INFO]: [AdvancedEnchantments] Successfully hooked into MythicMobs.
[23:08:11 INFO]: [AdvancedEnchantments] Successfully hooked into ProtocolLib.
[23:08:11 INFO]: [AdvancedEnchantments] Successfully hooked into SilkSpawners.
[23:08:11 INFO]: [AdvancedEnchantments] Successfully hooked into Essentials.
[23:08:11 INFO]: [AdvancedEnchantments] Loaded 7 armor sets.
[23:08:11 INFO]: [AdvancedEnchantments] Loaded 5 weapons.
[23:08:11 INFO]: [AdvancedEnchantments] Successfully hooked into economy plugin (Vault)
```

**Comment by Burchard36** [comment]
As @itsmemac said issue is fixed on that version :)
